TypeCobol/Tools/Options-Config/TypeCobolConfiguration.cs

```diff
@@ -35,6 +38,10 @@ public class TypeCobolConfiguration : ITypeCobolCheckOptions
         public const string DefaultLogFileName = "TypeCobol.CLI.log";
 
 #if EUROINFO_RULES
+        /// <summary>
+        /// The Instance of the Cpy Copy names file
+        /// </summary>
+        public CopyNameMapFile CpyCopyNamesMap { get; set; }
```

Comment:
`TypeCobolConfiguration` should deal with path only.

Reply:
done in e4234a0